### PR TITLE
[#28187] Add a Java gradle task to run validates runner tests on Prism.

### PR DESCRIPTION
**Pull request by lostluck**
* Add a Java gradle task to run validates runner tests on Prism.
* Filtered to green tests. Added notes about what's broken with them, along with links to pertinent issues.
* Broken tests that will terminate are in the sickbay list, to separate data, from configuration.
* Add log_level filtering to prism. It was easier to have the Java PrismRunner wrapper re-launch the binary all the time. Set Java default to warning log. Can be overridden by a `--prismLogLevel` flag. Replaces the recently added `--debug` flag. 
* Add a Gradle Action to run as Precommit.

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://github.com/apache/beam/blob/master/CONTRIBUTING.md#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI or the [workflows README](https://github.com/apache/beam/blob/master/.github/workflows/README.md) to see a list of phrases to trigger workflows.

### CODE REVIEW

**Comment by lostluck** [comment]
R: @damondouglas 

This should now be ready for review. The Github action was derived from the Flink PVR batch run, with a few unnecessary bits removed. It will be run once we merge this in.

I'll clean up the PreCommit, WsLint, and spotless once they come up.

File: runners/prism/java/src/main/java/org/apache/beam/runners/prism/PrismPipelineOptions.java [hunk 1]
```diff
@@ -59,4 +59,10 @@ public interface PrismPipelineOptions extends PortablePipelineOptions {
   String getIdleShutdownTimeout();
 
   void setIdleShutdownTimeout(String idleShutdownTimeout);
+
+  @Description("Sets the log level for Prism. Can be set to 'debug', 'info', 'warn', or 'error'.")
+  @Default.String("warn")
+  String getPrismLogLevel();
+
+  void setPrismLogLevel(String idleShutdownTimeout);
```
Comment (damondouglas) [inline runners/prism/java/src/main/java/org/apache/beam/runners/prism/PrismPipelineOptions.java:67]: Not: change `idleShutdownTimeout` to `value` or `prismLogLevel`.

**Review by damondouglas — APPROVED** [review]
Just one small comment.

File: runners/prism/java/src/main/java/org/apache/beam/runners/prism/PrismPipelineOptions.java (hunk 1 above)
Comment (lostluck) [reply runners/prism/java/src/main/java/org/apache/beam/runners/prism/PrismPipelineOptions.java]: Ah! Good catch. Thanks!

**Comment by lostluck** [comment]
Initial action run is here:

https://github.com/apache/beam/actions/runs/11507858474

**Pull request merged**